core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -25,13 +25,18 @@ public function get_script_configs() {
 				[
 					'handle' => 'elementor-packages-editor',
 					'src' => '{{ASSETS_URL}}/js/packages/editor{{MIN_SUFFIX}}.js',
-					'deps' => [ 'react', 'react-dom', 'elementor-packages-locations', 'elementor-packages-ui' ],
+					'deps' => [ 'react', 'react-dom', 'elementor-packages-locations', 'elementor-packages-ui', 'elementor-packages-store' ],
 				],
 				[
 					'handle' => 'elementor-packages-ui',
 					'src' => '{{ASSETS_URL}}/js/packages/ui{{MIN_SUFFIX}}.js',
 					'deps' => [ 'react', 'react-dom' ],
 				],
+				[
+					'handle' => 'elementor-packages-store',
+					'src' => '{{ASSETS_URL}}/js/packages/store{{MIN_SUFFIX}}.js',
```

Comment:
```suggestion
					'src' => '{{ASSETS_URL}}js/packages/store{{MIN_SUFFIX}}.js',
```

That was a mistake in all the other packages and it was fixed in my PR

Reply:
Done.